src/utlist.h

```diff
@@ -349,6 +349,26 @@ do {
   }                                                                                            \
 } while (0)
 
+#define LL_INSERT_INORDER(head,add,cmp)                                                        \
+  do {                                                                                         \
+    LDECLTYPE(head) _pos;                                                                      \
+    _CASTASGN(_pos, head);                                                                     \
+    LL_FIND_POS(_pos,add,cmp);                                                                 \
+    LL_APPEND_ELEM(head,_pos,add);                                                             \
+  } while (0)
+
+#define LL_FIND_POS(head,node,cmp)                                                             \
+  LL_FIND_POS2(head,node,cmp,next)                                                             \
```

Comment:
Nit: please remove the trailing backslash on this line (the last line of the macro definition).